From 6c6213edc0bde1bedcc4b364b7befcb156a220e4 Mon Sep 17 00:00:00 2001
Message-Id: <6c6213edc0bde1bedcc4b364b7befcb156a220e4.1228954349.git.christian@jaeger.mine.nu>
In-Reply-To: <030d49bbacadc03ef604478be7d0582bd2703cf7.1228954349.git.christian@jaeger.mine.nu>
References: <030d49bbacadc03ef604478be7d0582bd2703cf7.1228954349.git.christian@jaeger.mine.nu>
From: Christian Jaeger <christian@jaeger.mine.nu>
Date: Wed, 10 Dec 2008 23:11:06 +0100
Subject: [PATCH] Fix: initialize HASH_FOUND and LUKS_FOUND to ""

Instead of unset'ing them, which would make things fail with set -eu.

Signed-off-by: Christian Jaeger <christian@jaeger.mine.nu>
---
 debian/initramfs/cryptroot-hook |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/debian/initramfs/cryptroot-hook b/debian/initramfs/cryptroot-hook
index 815b133..9e25d09 100644
--- a/debian/initramfs/cryptroot-hook
+++ b/debian/initramfs/cryptroot-hook
@@ -200,8 +200,8 @@ get_device_opts() {
 	# We have all the basic options, let's go trough them
 	OPTIONS="target=$target,source=$source,key=$key"
 	local IFS=", "
-	unset HASH_FOUND
-	unset LUKS_FOUND
+	HASH_FOUND=""
+	LUKS_FOUND=""
 	for opt in $rootopts; do
 		case $opt in
 			cipher=*)
-- 
1.6.0.5

